app/client/src/components/pages/WaterbodyReport.js

```diff
@@ -1270,56 +1239,43 @@ function WaterbodyReport({ fullscreen }: Props) {
-                                        rel="noopener noreferrer"
-                                      >
-                                        {document.documentName}
-                                      </a>
-                                      <DynamicExitDisclaimer
-                                        url={document.documentURL}
-                                      />
-                                    </span>
-                                    {document.documentTypes.length && (
-                                      <>
-                                        <strong className='row-center'>&mdash;</strong>
-                                        <strong className='row-end'>
-                                          {document.documentTypes.map((type) => type.documentTypeCode).join(' / ')}
-                                        </strong>
-                                      </>
-                                    )}
-                                  </li>
-                                ))}
-                            </ul>
-                          </div>
-                        </>
-                      )}
+                    {documents.status === 'fetching' && <LoadingSpinner />}
+                    {documents.status === 'failure' && (
+                      <div css={modifiedErrorBoxStyles}>
+                        <p>{waterbodyReportError('Assessment')}</p>
+                      </div>
+                    )}
+                    {documents.status === 'success'
+                        && documents.data.length === 0 ? (
+                      <p>No documents are available</p>
+                    ) : (
+                      <>
+                        <em>Links below open in a new browser tab.</em>
+                        <table className="table">
+                          <thead>
+                            <tr>
+                              <th>Assessment</th>
+                              <th>Type</th>
+                            </tr>
+                          </thead>
+                          <tbody>
+                          {documents.data.map((document) => (
+                            <tr key={document.documentName}>
+                              <td>
+                                <a href={document.documentURL} target="_blank" rel="noopener noreferrer">
+                                  {document.documentName}
+                                </a>
+                                <DynamicExitDisclaimer url={document.documentURL} />
+                              </td>
+                              <td>
+                                {document.documentTypes.length && document.documentTypes[0].documentTypeCode}
+                              </td>
+                            </tr>
+                          ))}
+                          </tbody>
+                        </table>
+                      </>
+                    )}
```

Comment:
```suggestion
                    {documents.status === 'success' && (
                        <>
                          {documents.data.length === 0 ? (
                            <p>No documents are available</p>
                          ) : (
                            <>
                              <em>Links below open in a new browser tab.</em>
                              <table className="table">
                                <thead>
                                  <tr>
                                    <th>Assessment</th>
                                    <th>Type</th>
                                  </tr>
                                </thead>
                                <tbody>
                                  {documents.data.map((document) => (
                                    <tr key={document.documentName}>
                                      <td>
                                        <a
                                          href={document.documentURL}
                                          target="_blank"
                                          rel="noopener noreferrer"
                                        >
                                          {document.documentName}
                                        </a>
                                        <DynamicExitDisclaimer
                                          url={document.documentURL}
                                        />
                                      </td>
                                      <td>
                                        {document.documentTypes.length &&
                                          document.documentTypes[0]
                                            .documentTypeCode}
                                      </td>
                                    </tr>
                                  ))}
                                </tbody>
                              </table>
                            </>
                          )}
                        </>
                      )}
```

While testing I noticed that the table shows up while the loading spinner is visible. The above code makes it so the table does not show up until the fetch completes.

Reply:
Good catch!